src/olympia/lib/es/utils.py

```diff
@@ -47,7 +45,7 @@ def index_objects(ids, model, extract_func, index=None, transforms=None, objects
                 {
                     '_source': data,
                     '_id': ob.id,
-                    '_type': ob.get_mapping_type(),
+                    '_type': '_doc',
```

Comment:
Unfortunately, that doesn't work on existing indexes:
`{'type': 'illegal_argument_exception', 'reason': 'Rejecting mapping update to [addons-20210707143803] as the final mapping would have more than 1 type: [_doc, addons]'}`

Need to look if I can remove it entirely while keeping ES 6 compatibility. Might have to do something differently depending on the version to make the change transparent.

Reply:
https://github.com/mozilla/addons-server/pull/17457/commits/7060b61a3da3691aadf8e6c83a1133f54aad2138 should address this